packages/create-toolpad-app/utils/isFolderEmpty.ts

```diff
@@ -0,0 +1,66 @@
+// From https://github.com/vercel/next.js/blob/canary/packages/create-next-app/helpers/is-folder-empty.ts
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+
+export function isFolderEmpty(root: string, name: string): boolean {
+  const validFiles = [
```

Comment:
Are we really going to curate a list with random files? Why not just be strict? empty == no files?

Reply:
Fair enough, I imagined that we could rely on the list curated by https://github.com/vercel/next.js/blob/canary/packages/create-next-app/helpers/is-folder-empty.ts

Reply:
Should we leave it in, then?

Reply:
As you wish